.github/workflows/publish.yml

```diff
@@ -0,0 +1,70 @@
+name: Publish Docker Image
+
+on:
+  release:
+    types: [released]
+
+jobs:
+  docker-arm64:
+    runs-on: [self-hosted, linux, ARM64, docker-installed]
+    steps:
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Login to DockerHub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+      - name: Get Upload URL
+        id: create_release
+        shell: python
+        env:
+          tag_act: ${{ github.ref }}
+        run: |
+          import os
+          ref_tag = os.getenv('tag_act').split('/')[-1]
+          print(f"::set-output name=tag_name::{ref_tag[1:]}")
```

Comment:
Not sure whether it really makes sense to strip the "v" here, if it is added back manually in the next step regardless. It might also be easier to trigger this action on push of the tag instead of creation of the release?

This is a bit of a nitpick though, no reason to block the PR.